Swiftfin/Views/UserSignInView.swift

```diff
@@ -19,7 +19,19 @@ struct UserSignInView: View {
 	private var password: String = ""
 
 	var body: some View {
-		Form {
+		List {
+			#if !os(tvOS)
+				// DisclosureGroup not available on tvOS
+				if !viewModel.users.isEmpty {
```

Comment:
Please put these below the original text fields. This is dynamic, there may be no public users or many public users and will push down the main part of the view and mirrors the way local servers are handled.